src/styles/Colors.js

```diff
@@ -1,7 +1,7 @@
 export default Colors = {
-  primary: '#F4F4F4',
-  secondary: '#FFFFFF',
-  highlight: '#FF5553',
+  primary: '#89bdd3',
```

Comment:
I have to say I kind of like the other color scheme better haha. We could add some color by making each chart cell have a square with the three letter abbreviated name and a slick background color (maybe different color for each letter of the alphabet)